lib/cublas/libcublas.jl

```diff
@@ -3355,11 +3355,11 @@ end
     initialize_context()
     @ccall libcublas.cublasSgemvStridedBatched(handle::cublasHandle_t,
                                                trans::cublasOperation_t, m::Cint, n::Cint,
-                                               alpha::Ptr{Cfloat}, A::Ptr{Cfloat},
```

Comment:
These wrappers are auto-generated, so you should also update https://github.com/JuliaGPU/CUDA.jl/blob/master/res/wrap/cublas.toml

Reply:
I can't find these wrappers in that file. They only appear in this file. Probably that is why there was some type inconsistency.

Reply:
Yes, you need to add an entry. The purpose is that file is to add additional rewrites to the autogenerated headers.

Reply:
Ok. As I understand the structure, I only declare which variable's types I want to rewrite. Two questions:
1. Variable numbering begins from `1`?
2. What is the "default" type of a variable?

Reply:
> Variable numbering begins from `1`?

Yes



> What is the "default" type of a variable?

What's currently being generated based on the C headers. The TOML only provides overrides.